not found BLAS::BLAS target

```
-- Configuring done
CMake Error at src/laptools/CMakeLists.txt:9 (add_library):
  Target "laptoolslib" links to target "BLAS::BLAS" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at src/laptools/CMakeLists.txt:29 (add_executable):
  Target "seqcmaplap" links to target "BLAS::BLAS" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at src/laptools/CMakeLists.txt:29 (add_executable):
  Target "seqcmaplap" links to target "BLAS::BLAS" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at src/ug_cmaplap/CMakeLists.txt:51 (add_library):
  Target "fuglib" links to target "BLAS::BLAS" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


CMake Error at src/ug_cmaplap/CMakeLists.txt:98 (add_library):
  Target "fcmaplaplib" links to target "BLAS::BLAS" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?


CMake Error at src/ug_cmaplap/CMakeLists.txt:98 (add_library):
  Target "fcmaplaplib" links to target "BLAS::BLAS" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.18)
 project(CMAP-LAP CXX)
 
 set(CMAKE_CXX_STANDARD 11)

